ci-matrix-config.yml

```diff
@@ -0,0 +1,15 @@
+include:
+  - fqbn: esp32:esp32:featheresp32
+
```

Comment:
The idea of checking all projects alike is good and helpful.
What I dislike about this is that it is wholly unclear which test cases are executed for which fqbns. One needs to dive into arduino-devops and check the code to find out.

Reply:
True. We will anyhow document the arduino-devops project.
For a library all examples are run against all fqbn from the default list in arduino-devops. That is implicit, and we only have to explicitly add the exceptions to that matrix. 

It is possible to add a comment as well to this yaml, mentioning where the default list is taken from. But this will be pointed out when running the script:
`WARNING: Loading IFX default library fqbn list from "C:\path\arduino-devops\config\ci-config-matrix-ifx-lib.yml"`

Alternatively, you can add a "fqbn:" list specific for this project, instead of using the default from arduino-devops. That way it is explicit and clear for this project.